src/pages/search/ListingCard.tsx

```diff
@@ -16,7 +21,17 @@ const ListingCard = ({
       </div>
     </div>
     <CardBody>
-      <CardSubtitle className="small text-secondary">{homeType}</CardSubtitle>
+      <CardSubtitle className="small text-secondary justify-content-between row no-gutters">
+        {homeType}
+        {rating && <div style={{ marginTop: '-0.375rem' }}>
```

Comment:
Inline style tag here to fine-tune the vertical alignment of `StarRatings`. Does anyone see a way to get similar alignment with Bootstrap classes? (`mt-n1` and `mt-n2` go too low and too high, respectively). Alternately, should this be moved to `customStyles`?

Reply:
i think keeping the styles inline is fine, especially if it's just one style.  I've tried messing around with it but since the starts are absolutely positioned i dont think there's much we can do here.  solid fix 👍